sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -52,8 +57,9 @@
     private final DigitalTwinsServiceVersion serviceVersion;
     private final AzureDigitalTwinsAPIImpl protocolLayer;
     private static final Boolean includeModelDefinitionOnGet = true;
+    private JsonSerializer serializer = null;
 
-    DigitalTwinsAsyncClient(HttpPipeline pipeline, DigitalTwinsServiceVersion serviceVersion, String host) {
+    DigitalTwinsAsyncClient(HttpPipeline pipeline, DigitalTwinsServiceVersion serviceVersion, String host, JsonSerializer jsonSerializer) {
```

Comment:
I'm surprised the host is the 2rd parameter, but it is hard to articulate why. Is there parameter order guidance from team Azure SDK team?

Reply:
I double checked the configuration client, and they have it as "host, pipeline, serviceVersion", so I can switch us to match